ElixrTalentConnect/Extensions/SfSymbolSetup.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SfSymbolSetup.swift
```

Comment:
Can we give more appropriate name to the file? The file name not make any sense with the class name